community/cypher/cypher/src/main/scala/org/neo4j/cypher/CypherRuntime.scala

```diff
@@ -25,7 +25,8 @@ case object CypherRuntime extends CypherOptionCompanion[CypherRuntime] {
 
   case object default extends CypherRuntime("default")
   case object interpreted extends CypherRuntime("interpreted")
+  case object registerInterpreted extends CypherRuntime("enterprise-interpreted")
```

Comment:
Should the name 'register' be here? Perhaps while the public name is 'enterprise-interpreted', the internal name is 'sleipnir'?